webcrypto/subtle_crypto.go

```diff
@@ -35,9 +36,71 @@ type SubtleCrypto struct {
 // The `key` parameter should be a `CryptoKey` to be used for encryption.
 //
 // The `data` parameter should contain the data to be encryption.
-func (sc *SubtleCrypto) Encrypt(algorithm goja.Value, key goja.Value, data []byte) *goja.Promise {
-	// TODO: implementation
-	return nil
+func (sc *SubtleCrypto) Encrypt(algorithm, key, data goja.Value) *goja.Promise {
+	rt := sc.vu.Runtime()
+	promise, resolve, reject := sc.makeHandledPromise()
+
+	// 2.
```

Comment:
```suggestion
```
Are they sequential numbers for the various operations? If yes, then I'm for removing them if we don't have a specific reason for keeping them.

Reply:
Let's say I'm a new maintainer of this library, how can I learn this convention without asking? Isn't a direct link to the spec a better solution for this problem?

Reply:
@codebien that's a fair point. I've added a section to the README detailing the convention, and providing some guidance as a result 👍🏻 🙇🏻 

Side note: in practice, I wouldn't expect maintainers or contributors to jump in without having the specs open, and it was my experience reviewing Deno's implementation that you immediately understand what those comments are for if you look at the specs. It also already proved useful in previous PRs where reviewers were able to spot a certain step happened to early, thanks to those comments 😄